src/components/BRB/BRBPartnerBounties.tsx

```diff
@@ -115,6 +116,7 @@ const PartnerBountiesContainer = styled.div`
 const Header = styled.h3`
   font-size: 46px;
   font-weight: 400;
+  font-family: Glancyr;
```

Comment:
Fallback font should be mentioned: https://developer.mozilla.org/en-US/docs/Web/CSS/font-family